postprocessing-analysis/producer.py

```diff
@@ -7,5 +7,5 @@
 #test code for initial setup
```

Comment:
We planned to remove this file right..are we keeping it for now?

Reply:
good i'll remove it now. lets not have unnecessary things